tasks/src/unconditional-image-generation/data.ts

```diff
@@ -0,0 +1,65 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "The CIFAR-100 dataset consists of 60000 32x32 colour images in 100 classes, with 600 images per class.",
+			id:          "cifar100",
+		},
+		{
+			description: "Multiple images of celebrities, used for facial expression translation.",
+			id:          "CelebA",
+		},
+	],
+	demo: {
+		inputs: [
+			{
+				label:   "Choose model:",
+				content: "azuki",
+				type:    "text",
+			},
+			{
+				label:   "Select generation type:",
+				content: "default",
+				type:    "text",
+			},
+			{
+				label:   "Number of rows:",
+				content: "2",
+				type:    "text",
+			},
+		],
+		outputs: [
+			{
+				filename: "unconditional-image-generation/unconditional-image-generation-output.jpeg",
```

Comment:
I got it up on my local and it can't find the image path of the new image. 
<img width="767" alt="Ekran Resmi 2022-10-11 19 07 34" src="https://user-images.githubusercontent.com/53175384/195155805-0246efa8-cf1b-4a7d-862a-0950f987a2ba.png">
Also I think you don't necessarily need to pick model, these models usually have random seeds as inputs that later define noise to generate the image from, see this as an example: https://huggingface.co/spaces/huggan/crypto-gan 
the row parameter makes sense but maybe rename it as "number of images to generate"? Row doesn't sound clear in this context IMHO 🙂

Reply:
Oh let me fix that...